1.0rc1 tests failure with wcslib 4.24

Although marked as xfail, `test_radesys_defaults` and `test_radesys_defaults_full` fail with wcslib 4.24 (which is in the upcoming Debian Squeeze release):
```
____________________________ test_radesys_defaults _____________________________
   @pytest.mark.xfail(
        LooseVersion(_wcs.__version__) < LooseVersion("4.25"),
        reason="wcslib < 4.25")
    def test_radesys_defaults():
        w = _wcs.Wcsprm()
        w.ctype = ['RA---TAN', 'DEC--TAN']
        w.set()
>       assert w.radesys == "ICRS"
E       assert '' == 'ICRS'
E         + ICRS

astropy/wcs/tests/test_wcsprm.py:795: AssertionError
```
(test_radesys_defaults_full similar).

--- a/astropy/wcs/setup_package.py
+++ b/astropy/wcs/setup_package.py
@@ -230,6 +230,9 @@ def get_extensions():
         cfg['sources'].extend(join(wcslib_cpath, x) for x in wcslib_files)
         cfg['include_dirs'].append(wcslib_cpath)
     else:
+        wcsconfig_h_path = join(WCSROOT, 'include', 'wcsconfig.h')
+        if os.path.exists(wcsconfig_h_path):
+            os.unlink(wcsconfig_h_path)
         cfg.update(setup_helpers.pkg_config(['wcslib'], ['wcs']))
 
     astropy_wcs_files = [  # List of astropy.wcs files to compile

